core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactorySpec.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the AnswerClassificationResultObjectFactory.
+ */
+ 
+describe('Answer classification result object factory', function() {
+  var AnswerClassificationResultObjectFactory;
+  var DEFAULT_OUTCOME_CLASSIFICATION;
+  var oof;
+  
+  beforeEach(module('oppia'));
+
+  beforeEach(inject(function($injector) {
+    AnswerClassificationResultObjectFactory = $injector.get(
```

Comment:
Sorry for not catching it before, but let's name it as `acrof` instead of the full name.

Reply:
Hi ,@shubha1593 required changes Done!